test/core/security/credentials_test.cc

```diff
@@ -3603,87 +3593,10 @@ static void test_fake_call_credentials_compare_failure(void) {
 }
 
 int main(int argc, char** argv) {
+  ::testing::InitGoogleTest(&argc, argv);
```

Comment:
nit: prefer to not fully qualify, see https://abseil.io/tips/119

Reply:
Done